CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+
+### Greetings, contributor!
+
+Thank you for considering contributing to Remote Retro. We're happy to have you!
+
+### Types of contributions we're looking for
+
+The best way to contribute is to take a look at our current backlog and work on a story that you're most excited about. Please take a look at the [Project Management](https://github.com/stride-nyc/remote_retro#project-management) section of our Readme for viewing the backlog. Bug fixes and documentation improvements are also welcome.
+
+# Code of Conduct
+Please read our [Code of Conduct](https://github.com/stride-nyc/remote_retro/blob/master/CODE_OF_CONDUCT.md) before contributing.
+
+# First time contributing?
+Unsure where to begin contributing to Remote Retro? You can start by looking through [issues](https://github.com/stride-nyc/remote_retro/issues) that have the "beginner-friendly" or "help wanted" label.
+
+# Getting started
+* Before you start coding, please see the [Dev Environment Setup](https://github.com/stride-nyc/remote_retro#dev-environment-setup) and [Tests](https://github.com/stride-nyc/remote_retro#tests) sections of the Readme for information about getting set up and running the tests.
+* We require all contributors to sign a Contributor License Agreement before contributing. You will be prompted to electronically sign the individual CLA when you make a pull request.
```

Comment:
Is this the first time you make a PR or every time?

Reply:
great question. my hope is that it would be a one-time thing, or that at the very least it's configurable, but i also wouldn't consider it a blocker, as we need to get this in.